test/pages/trust-admin/wards/add.test.js

```diff
@@ -28,6 +28,7 @@ describe("/trust-admin/wards/add", () => {
     });
   });
 
+  /*** hospital drop down will be taken out skip test for now */
   describe("AddAWard", () => {
```

Comment:
Did you mean to skip this?

Reply:
Sorry, the test wasn't skipped but I forgot to remove the comment!
Will remove the comment now.

Reply:
Comment removed 🙂